fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/runtime/SecuredArchivePreparer.java

```diff
@@ -140,6 +140,14 @@ private static InputStream getKeycloakJsonFromClasspath(String resourceName) {
         return keycloakJson;
     }
 
+    private static InputStream loadAppArtifact(String appArtifact) {
+        InputStream result = Thread.currentThread().getContextClassLoader().getResourceAsStream("_bootstrap/" + appArtifact);
```

Comment:
debugger shows that TCCL works also for the jar execution. I just wanted to play it safe

Reply:
Nice, thanks!